File: SEIST/polyglot/fsharp/SeisReleaseContract.fs
namespace Seis.Polyglot

module SeisReleaseContract =
  let packagePath = "dist/seis-static.zip"
  let supportedLocales = [ "tr"; "en"; "fr"; "it"; "de"; "es"; "ar" ]
  let reducedMotionRequired = true
  let liveUploadRequiresTarget = true

  let supportsLocale locale =
    supportedLocales |> List.contains locale

File: SEIST/polyglot/scala/SeisReleaseContract.scala
object SeisReleaseContract {
  val packagePath: String = "dist/seis-static.zip"
  val locales: List[String] = List("tr", "en", "fr", "it", "de", "es", "ar")
  val reducedMotionRequired: Boolean = true
  val liveUploadRequiresTarget: Boolean = true

  def supportsLocale(locale: String): Boolean = locales.contains(locale)
}
